src/Filament/Widgets/Components.php

```diff
@@ -77,16 +78,15 @@ protected function buildToggleButton(Component $component): ToggleButtons
             ->inline()
             ->live()
             ->options(ComponentStatusEnum::class)
-            ->afterStateUpdated(function (ComponentStatusEnum $state) use ($component) {
-                return $component->update(['status' => $state]);
-            });
```

Comment:
On an unrelated note, this PR also makes use of arrow functions for better code consistency.